scripts/jwst/build_parent_sample.py

```diff
@@ -409,35 +357,22 @@ def main(output_dir: str, image_dir: str, survey: str, subsample: str, mag_cut:
         "--output_dir", type=str, help="The path to the output directory", default="."
     )
     parser.add_argument(
-        "--image_dir",
+        "--local_dir",
         type=str,
         help="The path to the temporary download directory",
-        default=".",
+        default="./jwst_data",
     )
     parser.add_argument(
-        "survey",
-        type=str,
-        help="Survey name. Currently supported survey keywords are: ceers-full,ngdeep,primer-uds,gds,gdn",
-    )
-    parser.add_argument(
-        "--subsample",
-        type=str,
-        default="all",
-        help="all or tiny. tiny downloads a random subset of 100 objects for testing purposes.",
-    )
-    parser.add_argument(
-        "--mag_cut",
-        type=float,
-        default=25.5,
-        help="Magnitude cut for the sample.",
+        "--max_workers",
+        type=int,
+        default=10,
+        help="Number of threads for parallel downloads and processing",
```

Comment:
Can we default to none and use `os.cpu_count()` as a default?

Reply:
os.cpu_count() is too much usually

Reply:
happy to switch default to 1 though
```suggestion
        "--max_workers",
        type=int,
        default=1,
        help="Number of threads for parallel downloads and processing",
```

Reply:
Yeah. Let's change the default to 1.